### ISSUE TITLE
Suggestions for Multiple App Installations

### ISSUE THREAD

**Issue by ais-developer** [thebaselab/codeapp#901]
I have installed multiple variants of the app. One thing that's missing is that the icons and flavouring of the app is same. It is often confusing which app variant I am using.

I am aware I can be using multiple windows but for the sake of working on multiple projects its much easier to install the app as a seperate variant.

It would be amazing if each variant could be distinguished by a theme and especially a different icon. Also include the ap icon some where in the actual app title window UI to show variant and enhance branding. 

Perhaps the same icon, but instead of blue rectangle, red circle, green triangle, yellow hexagon, etc.

This would require either for you guys to provide app icon images for different variants for edge cases for developers like myself or perhaps a dynamic icon color changing solution (meaning just keep the rectangle and change the color of line to red, blue, green, yellow..etc). 

Let me know what you guys think if this suggestion. Thank you so much so far for such an awesome job!

**Comment by bummoblizard** [comment]
I'm afraid releasing multiple apps on App Store is not the best idea. Not only it goes against App Store's rule, but it can be confusing for users. Currently, multiple windows are distinguished by the subtitle under the app's name. But we can take a look into offering icon variants.
<img width="398" alt="image" src="https://github.com/thebaselab/codeapp/assets/38398443/ae145cbb-99b5-4e78-a681-2a9acc516daf">

To distinguish among different installation of the app however, you can set different themes for them. Just out of curiosity, how did you install variants of the app? Did you build the app yourself?

**Comment by ais-developer** [comment]
Yes I built the app myself. That's why I was asking what your thoughts are on the matter. I agree we shouldn't have multiple variants of the app in app store, this is just a personal project I was working on (hopefully that's not okay). But yeah if you can provide me variants of the icon on the side, I would appreciate it very much :)

**Comment by bummoblizard** [comment]
> Yes I built the app myself. That's why I was asking what your thoughts are on the matter. I agree we shouldn't have multiple variants of the app in app store, this is just a personal project I was working on (hopefully that's okay). But yeah if you can provide me variants of the icon on the side, I would appreciate it very much :)

Unfortunately I lost the original file for the app icon. Since you are building the app yourself, you can change the icon into any you like in Xcode :)

**Comment by ais-developer** [comment]
No worries, I will try my best to come up with something half decent (fingers-crossed)
